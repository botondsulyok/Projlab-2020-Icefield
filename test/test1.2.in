loadmap testmap1
stats f1