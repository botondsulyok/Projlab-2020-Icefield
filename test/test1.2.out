id: f1
type: unstable
building: b1
snow: 3
pickable: i1
standers: e1
pullers: e1
masters: e1
neighbours: null
capacity: 5
broken: false

